status/backend.json

```
@@ -130,6 +130,18 @@
         "en-EN": "possible slowdowns"
       }
     },
+    "paypal": {
+      "is_visible": false,
+      "level": "warning",
+      "message": {
+        "it-IT": "TBD",
+        "en-EN": "TBD"
+      },
+      "badge": {
+        "it-IT": "TBD badge",
+        "en-EN": "TBD badge"
```

```suggestion
      "message": {
        "it-IT": "",
        "en-EN": ""
      },
      "badge": {
        "it-IT": "",
        "en-EN": ""
```

For consistency, perhaps it is better to leave them as empty string.